data_structures/binary_tree/binary_search_tree.py

```diff
@@ -141,6 +141,18 @@ def traversal_tree(self, traversal_function=None):
         else:
             return traversal_function(self.root)
 
+    def inorder(self, arr, Node):
+        if Node:
+            self.inorder(arr, Node.left)
+            arr.append(Node.value)
+            self.inorder(arr, Node.right)
+
+    def find_Kth_Smallest(self, k, Node):
```

Comment:
In this function, if you are doing an inorder traversal in a BST, then it will give you a non-decreasing order of elements. So does it requires an additional sorting??

Reply:
No sorting is not required you are right I will make that change.

Reply:
did all changes.

Reply:
Capital letters are not allowed in Python function names.